Add error message to remind user to use git meta push

shijinglu opened the pull request:
pre receive hook can reminder user of using `git meta push` when it deteches submodule sysnthetic branches are not updated along with the meta commit user is trying to push.

novalis commented:
I would like to make this message more prominent -- for instance, by adding some newlines before and after it.  Also, can you discuss how you tested it?

shijinglu commented:
> I would like to make this message more prominent -- for instance, by adding some newlines before and after it. Also, can you discuss how you tested it?

Thanks for the reminder, just updated and added new lines.

The bad meta update case is already tested in the test/synthetic-branch.js, so the test plan is that all existing tests should pass.  Adding an extra test for console.log seems not that critical.
